_config.yml

```diff
@@ -23,6 +23,12 @@ github_info:
 # See https://github.com/bryanbraun/anchorjs for more information.
 # anchor_js_targets: [h1, h2, h3, h4, h5, h6]
 
+# Enables Private Eye functionality. 
+# Comment out if you do not want this functionality.
```

Comment:
[The config will be inherited from the theme](https://jekyllrb.com/docs/themes/#pre-configuring-theme-gems40), so they'd have to explicitly disable rather than comment it out.

```suggestion
```